website/blog/2022-05-31-aluminium.md

```diff
@@ -0,0 +1,290 @@
+---
+author: Rikito Taniguchi
+title: Metals v0.11.6 - Aluminium
+authorURL: https://twitter.com/tanishiking25
+authorImageURL: https://github.com/tanishiking.png
+---
+
+We're happy to announce the release of Metals v0.11.6, which 
+
+- reduces memory usage in large projects
```

Comment:
Shouldn't this be in the TLDR ? That section is not that useful with `Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
give Metals a try!`

Reply:
That's a good idea! What do you think about https://github.com/scalameta/metals/pull/3957/commits/c1b589fffbcae5b756993a61d8d9c424890e0cee ?